documentation/inventory_file.md

```diff
@@ -214,11 +214,6 @@ nodes:
   - remote://my_aws_account
 ```
 
-- **[Inventory file version 2](inventory_file_v2.md#)**  
-Version 2 of the inventory file is experimental and might experience breaking changes in future releases.
-- **[Generating inventory files](inventory_file_generating.md)**  
```

Comment:
We should add a deprecation warning for removing this before we remove docs for it.

Reply:
Got it. I thought it was missed in #1309 but I'll add it back for now.